Code/Editor/Objects/ObjectManager.cpp

```diff
@@ -1327,7 +1331,7 @@ void CObjectManager::FindDisplayableObjects(DisplayContext& dc, [[maybe_unused]]
 
     pDispayedViewObjects->SetSerialNumber(m_visibilitySerialNumber); // update viewport to be latest serial number
 
-    const CCamera& camera = GetIEditor()->GetSystem()->GetViewCamera();
+    //const CCamera& camera = GetIEditor()->GetSystem()->GetViewCamera();
```

Comment:
All the commented code here is also safe to remove, this is legacy code anyways

Reply:
Addressed in 70b430082f